client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.tsx

```diff
@@ -550,9 +550,22 @@ export function getEnhancedTasks(
 								: translate( 'Upgrade plan' ),
 					};
 					break;
+				case 'verify_domain_email':
+					taskData = {
+						completed: ! isDomainEmailUnverified,
+						actionDispatch: () => {
+							recordTaskClickTracksEvent( flow, task.completed, task.id );
+							window.location.replace( task.calypso_path || '/me/account' );
```

Comment:
Does it make more sense to use `` `/domains/manage/${ siteSlug }` `` as the fallback?

Reply:
it does - thanks.